v2/tracing/tracing_test.go

```diff
@@ -0,0 +1,74 @@
+package tracing
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus"
+	. "github.com/onsi/gomega"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+)
+
+func Test_TracingMiddleware(t *testing.T) {
+	correlationId := "correlation-id"
+	timeToLive := time.Duration(10)
+	scheduledEnqueueTime := time.Now().Add(time.Duration(10))
+
+	// fake a remote span
+	initUTTracerProvider()
+	remoteCtx, remoteSpan := otel.Tracer("test-tracer").Start(context.Background(), "remote-span")
+	remoteSpan.End()
+
+	testCases := []struct {
+		description     string
+		message         *azservicebus.ReceivedMessage
+		carryRemoteSpan bool
+	}{
+		{
+			description:     "nil message, should start a new span without parent",
+			message:         nil,
+			carryRemoteSpan: false,
+		},
+		{
+			description: "should set span attributes from message",
```

Comment:
could you actually verify that the span attributes do have the expected values in your test?